lib/index.js

```diff
@@ -3,65 +3,45 @@
 const spawn = require('cross-spawn').spawn
 const path = require('path')
 const fs = require('fs')
+const ParseArgs = require('./parse-args')
 
-function EnvCmd (args) {
+function EnvCmd () {
   // Parse the args from the command line
-  const parsedArgs = ParseArgs(args)
-
-  // Attempt to open the provided file
-  let file
-  try {
-    file = fs.readFileSync(parsedArgs.envFilePath, { encoding: 'utf8' })
-  } catch (e) {
-    throw new Error(`Error! Could not find or read file at ${parsedArgs.envFilePath}`)
-  }
-
-  // Parse the env file string
-  const env = path.extname(parsedArgs.envFilePath).toLowerCase() === '.json'
-    ? Object.assign({}, process.env, require(parsedArgs.envFilePath))
-    : ParseEnvString(file)
-
-  // Execute the command with the given environment variables
-  if (parsedArgs.command) {
-    const proc = spawn(parsedArgs.command, parsedArgs.commandArgs, {
-      stdio: 'inherit',
-      env
-    })
-    process.on('SIGTERM', () => proc.kill('SIGTERM'))
-    proc.on('exit', process.exit)
-    return proc
-  }
-}
-
-function ParseArgs (args) {
-  if (args.length < 2) {
-    throw new Error('Error! Too few arguments passed to env-cmd.')
-  }
-
-  let envFilePath
-  let command
-  let commandArgs = args.slice()
-  while (commandArgs.length) {
-    const arg = commandArgs.shift()
-
-    // assume the first arg is the env file
-    if (!envFilePath) {
-      envFilePath = path.resolve(process.cwd(), arg)
-    } else {
-      command = arg
-      break
+  const parsedArgs = ParseArgs()
+  const files = parsedArgs.files
+  const command = parsedArgs.command
+
+  // For all files..
+  const env = files.reduce((env, file) => {
+    try {
+      // Attempt to open the provided file
+      const envFileText = fs.readFileSync(file, 'utf8')
+
+      // Parse the env file string
+      if (path.extname(file) === '.json') {
+        env = Object.assign(env, require(file))
+      } else {
+        const parsedEnv = ParseEnvString(envFileText)
+        env = Object.assign(env, parsedEnv)
+      }
+      return env
+    } catch (e) {
+      throw new Error(`Error! Could not find or read file at ${file}`)
     }
-  }
+  }, {})
 
-  return {
-    envFilePath,
-    command,
-    commandArgs
-  }
+  // Execute the command with the given environment variables
+  const proc = spawn(command[0], command.slice(1), {
+    stdio: 'inherit',
+    env
+  })
+  process.on('SIGTERM', () => proc.kill('SIGTERM'))
+  proc.on('exit', process.exit)
+  return proc
 }
 
 function StripComments (envString) {
-  const commentsRegex = /[ ]*(#.*$)/gim
+  const commentsRegex = /[ ]*((\#|\/\/).*$)/gim
```

Comment:
Like this, but it needs some tests. Those tests should include the comment on a line by itself and an inline comment.